libraries/botbuilder-azure-blobs/src/blobsStorage.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
The latest OSS@MS-blessed copyright headers have changed, I don't know if we want to update the existing files with such a change but I think it wouldn't hurt if we used it for new files.

```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```